translations/monero-GUI-guide_es.md

```diff
@@ -0,0 +1,331 @@
+
+# 7. Verificación de los binarios
+
+Debes asegurarte de que los ficheros que has descargado se corresponden con los ficheros oficiales. 
+Puedes utilizar [esta sencilla guía ilustrada](https://getmonero.org/resources/user-guides/verification-windows-beginner.html) para sistemas Windows.
+
+# 8. Problemas frecuentes y soluciones
+
++ **[[EN] He perdido (no estoy viendo) una transacción entrante en la interfaz gráfica (balance cero)](https://monero.stackexchange.com/questions/6640/i-am-missing-not-seeing-a-transaction-to-in-the-gui-zero-balance)**
++ **[[EN] Uso la interfaz gráfica y mi demonio ha dejado de arrancar](https://monero.stackexchange.com/questions/6825/i-am-using-the-gui-and-my-daemon-doesnt-start-anymore)**
++ **[[EN] La transacción está bloqueada como _pendiente_ en la interfaz gráfica](https://monero.stackexchange.com/questions/6649/transaction-stuck-as-pending-in-the-gui)**
++ **[[EN] La interfaz gráfica parece que tiene errores o se congela todo el rato](https://monero.stackexchange.com/questions/6651/my-gui-feels-buggy-freezes-all-the-time)**
++ **[[EN] Mi nombre contiene caracteres especiales (non-ASCII) (e.g. é, ø, â, Ö) y no puedo crear un monedero en la interfaz gráfica](https://monero.stackexchange.com/questions/6823/my-name-contains-a-special-non-ascii-character-e-g-%c3%a9-%c3%b8-%c3%a2-%c3%96-and-i-cant-c)**
++ **[[EN] La interfaz gráfica usa todo mi ancho de banda y no puedo navegar o usar otras aplicaciones que requieran conexión a internet](https://monero.stackexchange.com/questions/6653/the-gui-uses-all-my-bandwidth-and-i-cant-browse-anymore-or-use-another-applicat)**
++ **[[EN] ¿Cómo puedo mover la cadena de bloques (`data.mdb`) a un directorio diferente durante (o después) de la sincronización inicial sin perder los avances?](https://monero.stackexchange.com/questions/7225/how-do-i-move-the-blockchain-data-mdb-to-a-different-directory-during-or-afte)**
++ **[[EN] ¿Cómo puedo cambiar el idioma de las 25 palabras de la semilla mnemotécnica en la interfaz gráfica y en la terminal?](https://monero.stackexchange.com/questions/7373/how-do-i-change-the-language-of-the-25-word-mnemonic-seed-in-the-gui/)**
++ **[[EN] Mi cadena de bloques está bloqueada, ¿cómo la desatasco?](https://monero.stackexchange.com/questions/4462/my-blockchain-is-stuck-how-do-i-unstuck-it)**
++ **[[EN] Estoy usando un nodo remoto pero, ¿por qué sigue sincronizándose la cadena de bloques en la interfaz gráfica?](https://monero.stackexchange.com/questions/6324/using-remote-node-still-syncs-blockchain)**
++ **Utilizo una pantalla de alta resolución y la interfaz gráfica parece extremadamente pequeña**    
+Este problema será resuelto próximamente, pero se puede solucionar temporalmente en Windows: haz clic con botón derecho en `monero-wallet-gui.exe`, selecciona `Propiedades` --> `Compatibilidad`. En la opción de alta resolución (`high DPI`), cambia el valor de `Aplicación` a `Sistema` o viceversa.
```

Comment:
Not sure if this was on purpose but the last point is missing translation:

- TO DO: [How to add the blockchain manually](https://github.com/monero-ecosystem/monero-GUI-guide/tree/master/translations)

Reply:
Yep, it did not appear in the final English text so I omit that lines found in the placeholder. ;)